core/src/main/java/org/elasticsearch/common/io/stream/StreamInput.java

```diff
@@ -707,6 +708,13 @@ public ShapeBuilder readShape() throws IOException {
     }
 
     /**
+     * Reads a {@link SmoothingModel} from the current stream
+     */
+    public SmoothingModel readSmoothingModel() throws IOException {
```

Comment:
Can we name this method `readPhraseSuggesterSmoothingModel()` so it doesn't confuse with other smoothing models we might have in the future (e.g. for a interpolating pipeline aggregator)?